### PR TITLE
Fix cross-refs for new Bikeshed changes

### LINKED ISSUE #3752: WebGPU bikeshed error:  LINE ~6751: No 'dfn' refs found for 'serializable objects' that are marked for export.
The bulid-validate-publish flow is failing.
E.g. https://github.com/gpuweb/gpuweb/actions/runs/3943724824/jobs/6748888303

make: *** Waiting for unfinished jobs....
LINE ~6751: No 'dfn' refs found for 'serializable objects' that are marked for export.
[=serializable objects=]
 ✘  Did not generate, due to fatal errors
make[1]: *** [Makefile:10: index.pre.html] Error 2

I think the problem is in spec/index.bs:

$ grep -n "serializable ob" index.bs 
6751:{{GPUPipelineError}} objects are [=serializable objects=].

### PR DESCRIPTION
**Pull request by kainino0x**
Fixes #3752

### CODE REVIEW

File: wgsl/index.bs [hunk 1]
```diff
@@ -11451,18 +11463,18 @@ See [[#builtin-inputs-outputs]] for how to declare a built-in value.
       <td style="width:50%">Sample coverage mask for the current fragment.
          It contains a bitmask indicating which samples in this fragment are covered
          by the primitive being rendered.
-         <br>See [[WebGPU#sample-masking|WebGPU &sect; Sample Masking]].
+         <br>See [[WebGPU#sample-masking]].
```
Comment (kainino0x) [inline wgsl/index.bs:11466]: Note this changes
> WebGPU &sect; Sample Masking

to
> _WebGPU_ § 23.3.11 Sample Masking

File: wgsl/index.bs [hunk 2]
```diff
@@ -3339,7 +3353,7 @@ If a reference or pointer access is out of bounds, an
 [=Load Rule|Loads=] from an invalid reference return one of:
     * when the [=originating variable=] is a [=uniform buffer=] or a [=storage buffer=],
         the value from any [=memory locations|memory location(s)=]
-        of the [[WebGPU#buffers|WebGPU buffer]] bound to the originating variable
+        of the {{GPUBuffer}} bound to the originating variable
```
Comment (dneto0) [inline wgsl/index.bs:3356]: nit: I'd prefer this to say   "WebGPU {{GPUBuffer}}" so readers don't go looking in this doc.

File: wgsl/index.bs [hunk 3]
```diff
@@ -3349,7 +3363,7 @@ If a reference or pointer access is out of bounds, an
 [=statement/assignment|Stores=] to an invalid reference may do one of:
     * when the [=originating variable=] is a [=uniform buffer=] or a [=storage buffer=],
         store the value to any [=memory locations|memory location(s)=] of the
-        [[WebGPU#buffers|WebGPU buffer]] bound to the [=originating variable=]
+        {{GPUBuffer}} bound to the [=originating variable=]
```
Comment (dneto0) [inline wgsl/index.bs:3366]: Same here please write "WebGPU {{GPUBuffer}}"

File: wgsl/index.bs [hunk 4]
```diff
@@ -6275,7 +6289,7 @@ See [[#sync-builtin-functions]].
           * Otherwise, 0.
        * If |e1| is the most negative value in |T|, and |e2| is -1:
           * It is a [=shader-creation error=] if |e1| and |e2| are [=const-expressions=].
-          * It is a [=pipeline-creation error=] if |e1| and |e2| are [=override-expressions].
+          * It is a [=pipeline-creation error=] if |e1| and |e2| are [=override-expressions=].
```
Comment (dneto0) [inline wgsl/index.bs:6292]: ooh, nice catch.

**Review by dneto0 — APPROVED** [review]
LGTM with two tiny nits.

**Comment by dneto0** [comment]
And thanks!

**Pull request merged**